tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -860,9 +860,9 @@ platform_tests/test_reboot.py::test_fast_reboot:
 
 platform_tests/test_reboot.py::test_soft_reboot:
   skip:
-    reason: "Skip test_soft_reboot for m0/mx"
+    reason: "Skip test_soft_reboot for m0/mx and test is supported only on S6100 hwsku"
     conditions:
-      - "topo_type in ['m0', 'mx']"
+      - "topo_type in ['m0', 'mx'] or hwsku not in ['Force10-S6100']"
```

Comment:
Just `hwsku not in ['Force10-S6100']` is sufficient.

Reply:
@vaibhavhd Why should we remove earlier skip condition?

Reply:
@vvolam the first `OR` condition i think will return True for S6100 which means, it will skip 6100?

Reply:
I agree with Prince. So, my comment is not non-blocking anymore, it is blocking :D

Removing m0, mx is acceptable as s6100 are not used as mgmt devices anyway.
So if you skip for everything other than s6100, then M0 devices are automatically skipped.

Reply:
>  first OR condition i think will return True for S6100 which means,

this will not happen as s6100 are not used as m0,mx. So, first OR condition should return false for s6100, right?

Reply:
agree.